docs/rpc/rpc-spans.md

```diff
@@ -79,14 +78,30 @@ Examples of span names:
   `MyServiceReference.ICalculator/Add` reported by the client for .NET WCF calls
 - `MyServiceWithNoPackage/theMethod`
 
-### Common attributes
+#### Service name
```

Comment:
```suggestion
### Service name
```

toc should be also rebuilt for this change

Reply:
Fixed